adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -1608,14 +1608,14 @@ func (swagger *MgwSwagger) PopulateSwaggerFromAPIYaml(apiData APIYaml, apiType s
 					Verb:        operation.Verb,
 					Description: operation.Description,
 				}
-				if operation.OperationProxyMapping.Target.Target == "" {
```

Comment:
The way I would prefer is having this if block as `!= nil ` to improve readabillity (my personal opinion). But this also works